### ISSUE TITLE
Add permission to read `/run/user/$id/at-spi/bus` by default

### ISSUE THREAD

**Issue by fiftydinar** [pkgforge-dev/DeaDBeeF-AppImage#9]
That should fix this error in DeadBeeF:
```
(deadbeef:2): dbind-WARNING **: 07:29:55.145: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/1000/at-spi/bus: File o directory non esistente
```

I don't know if it also needs write permissions.
